src/Medidata.ZipkinTracer.Core.Collector/SpanCollector.cs

```diff
@@ -9,30 +9,27 @@ public class SpanCollector
         internal static BlockingCollection<Span> spanQueue;
 
         internal SpanProcessor spanProcessor;
-        private IClientProvider clientProvider;
 
         private static SpanCollector instance;
 
-        public static SpanCollector GetInstance(IClientProvider clientProvider, int maxProcessorBatchSize, ILog logger)
+        public static SpanCollector GetInstance(string zipkinServer, int zipkinPort, int maxProcessorBatchSize, ILog logger)
```

Comment:
If this is on the http stack, you can simply use the Uri type which incorporates the protocol (http), hostname and the port as well. I think it can be left to the caller to construct the zipkin server url (optionally with detecting that the url contains the proper zipkin trace path ('/api/v1/spans') in it - and if not, just add this to the end).